api/nakadi-event-bus-api.yaml

```diff
@@ -2,18 +2,73 @@ swagger: '2.0'
 info:
   title: Nakadi Event Bus API Definition
   description: |
-    -------------------------------
-    Nakadi Event Bus API Definition
+    -----------
+    Definitions
+    -----------
+    Nakadi at its core aims at being a generic and content-agnostic event broker with a convenient API.
+    In doing this, Nakadi abstracts away, as much as possible, details of the backing messaging
+    infrastructure. The single currently supported messaging middleware is Kafka (Kinesis is planned
+    for the future).
+
+    In Nakadi every Event has an EventType, and a **stream** of Events is exposed for each registered
+    EventType.
+
+    An EventType defines properties relevant for the operation of its associated stream, namely:
+
+    * The **schema** of the Event of this EventType (*TODO: define shcema*)
+
+    * The expected **validation** and **enrichment** procedures upon reception of an Event.
+      (*TODO: define validation and enrichment*)
+
+    * The **ordering** expectations of Events in this stream. (*TODO: define ordering not in terms of
+    partitioning*)
+
+    Except for defined enrichment rules, Nakadi will never manipulate the content of any Event.
+
+    Clients of Nakadi can be grouped in 2 categories: **Application owners** and **Clients** (clients in turn
+    are both **Producers** and **Consumers** of Events). Application owners interact with Nakadi via the
+    **Schema Registry API** for the definition of EventTypes, while Clients via the **Stream API** for
+    submission and reception of Events. A low level **Topic API** is available, providing full control and
+    responsibility of position tracking and partitioning (and therefore ordering) of to the Clients.
+
+    The consumption of Events proceeds via the establishment of a named **Subscription** to an EventType.
+    Subscriptions are persitent relationships from an Application (which might have several instances) and the
+    stream of an EventType, whose consumption tracking is managed by Nakadi, freeing Consumers from any
+    responsibility in tracking the current position on a Stream.
+
+
+
+    Scope and status of this document
     -------------------------------
 
-    * Low-level API
-    * Stantardised event format (see #/definitions/Event)
-    * Partition strategy: event store will infer the partition from the `partitioning_key` in the message body
+    The present API specification is in **draft** state and is subject to change.
```

Comment:
This point is not quite clear for me. If the API will change we should not put the first version on review then. If it will be extended (not changed) then it's ok.